packages/ui-extensions/src/surfaces/admin/components/Select/index.ts

```diff
@@ -0,0 +1,7 @@
+export type {
```

Comment:
What is this file for?

Reply:
hum... Let's keep this consistent with Checkout then. I'll remove the index files.